model_api/cpp/models/include/models/detection_model_yolo.h

```diff
@@ -83,3 +83,24 @@ class ModelYolo : public DetectionModelExt {
     std::vector<int64_t> presetMasks;
     ov::Layout yoloRegionLayout = "NCHW";
 };
+
+class YOLOv5 : public DetectionModelExt {
+    // Reimplementation of ultralytics.YOLO
+    void prepareInputsOutputs(std::shared_ptr<ov::Model>& model) override;
+    void updateModelInfo() override;
+    void init_from_config(const ov::AnyMap& top_priority, const ov::AnyMap& mid_priority);
```

Comment:
Do we use CamelCase for methods names, or there is no style preferences?

Reply:
The misalignment comes form OMZ. It used to be CamelCase. But before relocation we started moving to snake_case to align with OpenVINO, and it was not finished: https://github.com/openvinotoolkit/open_model_zoo/pull/3437. I define new functions in new style.